src/styles/blocks/card.scss

```
@@ -0,0 +1,34 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 50px auto;
+  align-items: center;
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent;
+    margin-bottom: 4px;
+  }
+
+  &__image {
+    margin-bottom: 40px;
```

```suggestion
    margin: 0 auto 40px;
```

To center your image inside card